packages/core/src/builder.class.ts

```diff
@@ -460,6 +469,26 @@ export type GetContentOptions = AllowEnrich & {
    * @deprecated use `enrich` instead
    */
   noTraverse?: boolean;
+
+  /**
+   * Property to order results by.
+   * Use 1 for ascending and -1 for descending.
+   *
+   * The key is what you're sorting on, so the following example sorts by createdDate
+   * and because the value is 1, the sort is ascending.
+   *
+   * @example
+   * ```
+   * createdDate: 1
+   * ```
+   */
+  sort?: { [key: string]: 1 | -1 };
```

Comment:
@murtaza-builder As you discussed with me, I don't actually see any logic supporting `sort` in the gen1 SDK. We should either:
- add the logic, OR
- remove this documentation

I would probably remove this from docs since no one's been complaining about the lack of sorting in gen1. They likely were passing `sort: JSON.stringify({})` in the `options.options` field here:

https://github.com/BuilderIO/builder/blob/fdcaad5c2f1598c99479c354eeee390f4e55b731/packages/core/src/builder.class.ts#L444-L447